bin/elasticity-flexture-L0/2_bending_of_lithosphere_under_line_loads.ipynb

```diff
@@ -0,0 +1,405 @@
+{
```

Comment:
<p>Is there a source credit for this image?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geodynamics-hubzero/gem-notebooks/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>